### ISSUE TITLE
examples/cnn_mnist dose not work well

### ISSUE THREAD

**Issue by loujt1984** [vlfeat/matconvnet#468]
I'm a new learner of MatConvNet. When I use cnn_mnist train the model on mnist data,I have got models 'net-epoch-1.mat' to 'net-epoch-20.mat'. But when I use res = vl_simplenn(net, im) to test the model, where im is an image from mnist, there has something wrong. And I find that the net.layers{end}.type is 'softmaxloss' in my trained model,which is different from 'imagenet-vgg-f.mat' downloaded from website. Can you give me some help? thank you.

**Comment by afuller187** [comment]
Edit the vl_simplenn file with this line at the top of the code:

net.layers(end) = []

This removes the loss layers form your net.
